rcswitch/plugin.yaml

```diff
@@ -5,82 +5,94 @@ plugin:
+        valid_max: 40
+        description:
+            de: "GPIO-Pin an dem der 433 MHz Sender angeschlossen ist"
+            en: "GPIO-Pin to which the 433 MHz sender is connected to"
+            
     rcswitch_sendDuration:
         type: num
-        default: 0.5
+        default: 0.1
         valid_min: 0.0
         valid_max: 2.0
         description:
-            de: "Minimale Zeit zwischen dem Senden von Befehlen in Sekundene"
-            en: "Minimum time in s between sending commands"
-
-    rcswitch_host:
-        type: ip
-        description:
-            de: "IP oder HOSTNAME des rcswitch hosts, falls das System nicht auf dieser SmartHomeNG-Maschine läuft. \
-                Achtung: Auf dem Remote Host muss ein SSH Server installiert sein."
-            en: "IP or HOSTNAME of rcswitch host, in case rcswitch is running on a remote machine, if IP the IPv4 address has to be \
-                 specified. Note: a SSH server has to be installed on the remote machine."
-
-    rcswitch_user:
-        type: str
-        description:
-            de: "User auf dem Remote Host"
-            en: "user at remote host"
-
-    rcswitch_password:
-        type: str
-        description:
-            de: "Passwort auf dem Remote Host"
-            en: "password for user at remote host"
+            de: "Minimale Zeit in Sekunden zwischen dem Senden von verschiedenen Befehlen"
+            en: "Minimum time in s between sending of different commands"
 
 
 item_attributes:
     # Definition of item attributes defined by this plugin
-    rc_device:
+    rc_SystemCode:
         type: str
-        valid_list: ['1','2','3','4','5','a','b','c','d','e','A','B','C','D','E']
         description:
-            de: "Nummer (oder Buchstabe) des Device"
-            en: "Number or letter or the device"
-
-    rc_code:
+            de: "SystemCode des Gerätes - Muss in der Form von 5 binären Digits angegeben werden [00000 - 11111]"
+            en: "SystemCode of device - Must be 5 binary digits [00000 - 11111]"
+    
+    rc_ButtonCode:
         type: str
+        valid_list:
+            - 'A'
+            - 'B'
+            - 'C'
+            - 'D'
+            - 'E'
+            - '10000'
+            - '01000'
+            - '00100'
+            - '00010'
+            - '00001'  
         description:
-            de: "Code des Devices - Muss in der Form von 5 binären Digits angegeben werden [00000 - 11111]"
-            en: "code of device - Must be 5 binary digits [00000 - 11111]"
-
+            de: "ButtonCode (oder Buchstabe) des Gerätes"
+            en: "ButtonCode (or letter) of the device"
 
+    
 item_structs: NONE
-  # Definition of item-structure templates for this plugin
+    # Definition of item-structure templates for this plugin
 
 plugin_functions: NONE
     # Definition of plugin functions defined by this plugin
 
 logic_parameters: NONE
     # Definition of logic parameters defined by this plugin
 
+webif_pagelength:
```

Comment:
Es gibt keinen Hauptabschnitt webif_pagelength in den Metadaten. 

Ist hier evtl. ein Arbeitsstand von OnkelAndy bzgl. des Webinterfaces an falscher Stelle eingeflossen? 
Das war in dem Arbeitsstand ein Parameter. Den gibt es aber in den Metatdaten Dateien (plugin.yaml) nicht mehr. Bitte den Abschnitt löschen.

Reply:
@msinn ich bin etwas verwirrt...
Anbei ein Bild vom aktuellen Master.

![grafik](https://user-images.githubusercontent.com/26808969/223058416-4911d509-e745-4936-a85e-25201e86196a.png)


![grafik](https://user-images.githubusercontent.com/26808969/223057984-930ba8ad-e3d7-494e-bdd0-d229615336c6.png)

Reply:
Ja, das muss aus dem sample_plugin wieder raus. Das ist im develop und ab dem nächsten Release (in ca. 1 bis 2 Wochen) ein globaler Parameter der nicht in den einzelnen Plugins definiert werden muss (wie INSTANCE bei Multti-Instance Plugins)

Reply:
Übrigens: Im tools Verzeichnis im develop Branch gibt es das Tool check_plugin.py, welches eine ganze Reihe von formalen Prüfungen durchführt und Verbesserungsvorschläge macht. Damit kannst Du Dein Plugin schon mal checken, bevor Du einen PR stellst.

Reply:
@msinn cooler checker 👍 

![grafik](https://user-images.githubusercontent.com/26808969/223090915-d8183684-d6dc-4d88-8be6-d6509a64e64a.png)

Ich würde hier noch einen Zeilenumbruch einbauen zwecks Lesbarkeit...

![grafik](https://user-images.githubusercontent.com/26808969/223091374-2cdb38d0-7145-438e-b2d5-46a5c56f3a3b.png)

Reply:
erledigt in PR für `rcs1000n`